### PR TITLE
Translation Updates

### PR DESCRIPTION
**Pull request by bransteitterbr**
Continuation of https://github.com/episphere/connect/issues/915

### CODE REVIEW

File: js/pages/agreements.js [hunk 1]
```diff
@@ -280,10 +280,10 @@ export const renderAgreements = async () => {
                                 <div class="row">
                                     <div class="col" style="padding-left: 30px; padding-right:30px;">
                                         <div class="row"  style="border:1px solid lightgrey; border-radius:5px;">
-                                            <div class="col-md-2">
+                                            <div class="col-md-2" data-i18n="agreements.surveyIcon">
                                                 <i class="fab fa-wpforms d-none d-md-block" title="Survey Icon" style="margin-left:10px; font-size:50px;color:#c2af7f;"></i>
                                             </div>
-                                            <div class="col-md-10 cnnsentBodyFont2">
+                                            <div class="col-md-10 cnnsentBodyFont2" data-i18n="agreements.stopSharinRecords">
```
Comment (amber-emmes) [inline js/pages/agreements.js:286]: This should be `Sharing`, right?

File: js/pages/healthCareProvider.js [hunk 2]
```diff
@@ -72,116 +72,116 @@ export const healthCareProvider = (siteId) => {
...
-    template += `
+    template += translateHTML(`
                     </select>
                 </label>
             </div>
             <div class="row">
-            <span class="required">*Required</span>
+            <span class="required" data-i18n="provider.required">*Required</span>
                 <div class="ml-auto">
-                <button style="display:none;" id="openModal" class="btn btn-primary save-data consentNextButton" data-toggle="modal" data-target="#HealthProviderModal">Submit</button>
```
Comment (amber-emmes) [inline js/pages/healthCareProvider.js:91]: I'm a little unclear on the purpose of these double Submit buttons. I do see that this is in the source file, but is this correct?

File: js/pages/agreements.js (hunk 1 above)
Comment (bransteitterbr) [reply js/pages/agreements.js:286]: That is actually the way the translation team tagged it :) but I will change it.

**Review by amber-emmes — APPROVED** [review]
Looks good!

**Pull request merged**